docs/projects/cn0363/index.rst

```diff
@@ -0,0 +1,285 @@
+.. _cn0363:
+
+CN0363 HDL project
+=================================================================================
+
+Overview
+---------------------------------------------------------------------------------
+
+The :adi:`EVAL-CN0363-PMDZ` is a dual channel colorimeter featuring a modulated
+light source transmitter, programmable gain transimpedance amplifiers on each
+channel, and a very low noise, 24-bit sigma delta ADC. The output of the ADC
+connects via a standard PMOD connector to a FPGA. The FPGA takes the sampled
+data from the ADC and implements a synchronous detection algorithm. By using
+modulated light and digital synchronous detection, rather than a constant (dc)
+source, the system strongly rejects any noise sources at frequencies other than
+the modulation frequency, providing excellent accuracy.
+
+The dual channel circuit measures the ratio of light absorbed by the liquids in
+the sample and reference containers at three different wavelengths. This forms
+the basis of many chemical analysis and environmental monitoring instruments
+used to measure concentrations and characterize materials through absorption
+spectroscopy.
+
+The :adi:`EVAL-CN0363-PMDZ` HDL reference design is based on the Analog Devices
```

Comment:
`https://www.analog.com/EVAL-CN0363-PMDZ` doesn't exist.
The website now redirects to `https://www.analog.com/en/search.html?q=EVAL-CN0363-PMDZ` when 404.
The correct seems to be
`https://www.analog.com/en/resources/reference-designs/circuits-from-the-lab/CN0363.html`,
that means
```:adi:`CN0363` ```

Reply:
Done in V2.